### ISSUE TITLE
Attachments not uploading

### ISSUE THREAD

**Issue by GWPublisher** [Tizra/Tizra-Customer-Tracker#429]
We have a number of products that are having issues uploading attachments. When we try to upload anything to these products, it does not upload, but goes to the Admin homepage. Two products that are having this issue are Modern Metalworking and Housing and Interior Design OIR. We are not sure if this is completely an issue with every product, but it probably has. We have been having issues with other titles over the last week.

**Comment by daviddurand** [comment]
This was another file capacity issue -- in this case, there seem to have been several alternate ways to fix the directory where uploads take place -- and the system made a different choice than it had previously. I have added more configurations specifying the proper disk for temporary data, and have tested (again) that they are now being written to the correct place.

I suspect that an over-zealous security policy changed the temp directory automatically again after the restart.

We will check this again at the next restart, just in case, but I think the additional changes that I made should now have fixed it for good.
